apps/hashdotai/integrations/integration-list.json

```diff
@@ -0,0 +1,142 @@
+[
+  {
+    "name": "Linear",
+    "logoUrl": "https://dummyimage.com/160x160/000/fff&text=Linear",
+    "summaryHtml": "Issues, users, organizations, projects, milestones, and much more",
+    "connectUrl": "/integrations/linear/connect",
+    "aboutParas": [
+      {
+        "header": "Realtime two-way sync",
+        "copyHtml": "Create, update or delete data in either Linear or HASH while keeping them in sync.",
+        "iconUrl": "https://dummyimage.com/160x160/ddd/000000&text=two-way"
+      },
+      {
+        "header": "Jointly maintained by Linear & HASH",
+        "copyHtml": "<div><strong>This integration is officially supported by both Linear and HASH.</strong></div><div>It is jointly maintained to ensure robust, reliable and realtime operation.</div>",
+        "iconUrl": "https://dummyimage.com/160x160/ddd/000000&text=double-tick"
+      }
+    ],
+    "availability": "Available",
+    "frequency": "Real-time",
+    "supportedTypes": [
+      {
+        "title": "Linear Project",
+        "iconUrl": "https://dummyimage.com/160x160/ddd/000000&text=Project",
+        "properties": [
+          {
+            "title": "Logo",
+            "description": "The workspace logo (square)",
+            "iconUrl": "https://dummyimage.com/160x160/ddd/000000&text=Logo",
+            "type": "image",
```

Comment:
@CiaranMn Pre-specifying these, versus providing the full type path/URL, and inferring the name/icon from that? Presume this is just a stopgap solution?

Reply:
Yes, this is a stopgap solution – I imagine this whole thing will actually be composed of multiple types, entities etc that need to be stitched together for the same data